cli/commands/pack.go

```diff
@@ -81,12 +81,13 @@ func parsePackageDeps(deps []string, depsFile string) (common.PackDependencies,
 	}
 
 	if depsFile == "" && len(deps) == 0 {
-		defaultPackeDepsFilePath := filepath.Join(ctx.Project.Path, defaultPackageDepsFile)
-		if _, err := os.Stat(defaultPackeDepsFilePath); err != nil {
-			return nil, fmt.Errorf("Failed to use default package dependencies file: %s", err)
+		defaultPackDepsFilePath := filepath.Join(ctx.Project.Path, defaultPackageDepsFile)
+		if _, err := os.Stat(defaultPackDepsFilePath); err != nil {
```

Comment:
Here we need to separate two cases: 
* `os.IsNotExists(err)` - file doesn't exist, used didn't specify some deps, no warns, no errors;
* `err != nil` - user specified a file, but we can't use it - error;
* `err == nil` - file exists, and we can use it.